st2common/st2common/policies/concurrency.py

```diff
@@ -35,6 +35,6 @@ def _get_status_for_policy_action(self, action):
         if action == 'delay':
             status = action_constants.LIVEACTION_STATUS_DELAYED
         elif action == 'cancel':
-            status = action_constants.LIVEACTION_STATUS_CANCELED
+            status = action_constants.LIVEACTION_STATUS_CANCELING
```

Comment:
Just trying to understand this and thinking out loud - how will this work and affect non-mistral tasks?

For mistral actions this will eventually be changed to canceled once Mistral updates internal task status and once we query it, right? How about non-mistral actions?